modules/@ergonode/media/src/components/Forms/ResourceRelationsForm.vue

```diff
@@ -28,25 +28,17 @@
             </FormSection>
         </template>
     </Form>
-    <div
-        class="products-relations-placeholder"
-        v-else>
-        <ListPlaceholder
-            :orientation="horizontalOrientation"
-            title="Nothing to see here"
-            subtitle="Here you can see information about relations in products"
-            :bg-url="require('@UI/assets/images/placeholders/comments.svg')" />
-    </div>
+    <TabBarNoDataPlaceholer
+        v-else
+        title="Nothing to see here"
+        subtitle="Here you can see information about relations in products" />
```

Comment:
in this module, you already use `i18n` translations. This text should be replaced by the placeholder `$t()`.